router/index.js

```diff
@@ -5,43 +5,43 @@ const express = require('express');
 const router = express.Router();
 
 // Register route
-router.get('/register', (req, res) => {
+router.get('/register', (req, res, next) => {
   res.sendStatus(200);
   // res.send('Register User');
```

Comment:
Like stated above, the use of comments helps improve the readability of the code.